src/controllers/basic-messages/BasicMessageController.ts

```diff
@@ -1,11 +1,12 @@
 import type { BasicMessageRecord, BasicMessageStorageProps } from '@aries-framework/core'
+import type { RecordId } from '../examples.js'
 
 import { Agent, RecordNotFoundError } from '@aries-framework/core'
 import { Body, Controller, Example, Get, Path, Post, Route, Tags, Response } from 'tsoa'
 import { injectable } from 'tsyringe'
 
-import { BasicMessageRecordExample, RecordId } from '../examples'
```

Comment:
why is this now being imported separately on lines 2 and 8?

Reply:
I've combined into single imports and added an eslint rule to warn on duplicate import